### ISSUE TITLE
how to change background color to the original one?

### ISSUE THREAD

**Issue by LukeLin** [Hyzeta/vscode-theme-github-light#1]
Is there any way change background color to the original one? The new version's background color is too light for me. I'd like the older one.

**Comment by yunlingz** [comment]
@LukeLin In version 3.1.X, I used a gray background. The reason is as you said: the pure white background is a bit dazzling. The gray background looks great on my Mac, but not long ago I found the gray background on some screens seems to be very strange (too black compare to Retina screen). This is not the case with a pure white background (as long as you don't adjust the brightness of the display too high). So in version 3.2, I switched back to a pure white background.

  Just publish v3.3.0. In this version, both white background and gray background are available now.
